packages/use-contractkit/src/connectors/useMetaMaskConnector.ts

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
```

Comment:
This hook fixes the infinite loop when one gets an error while connecting to meta mask.